protocol/x/clob/keeper/process_operations_test.go

```diff
@@ -2129,6 +2129,810 @@ func TestProcessProposerOperations(t *testing.T) {
 			},
 			expectedError: types.ErrDeleveragingIsFinalSettlementFlagMismatch,
 		},
+		"Succeeds with maker order that has a router fee": {
```

Comment:
nice job covering the cases - If i'm being nitpicky maybe next time we include these cases in the PR message